### ISSUE TITLE
Contain-facebook breaks multi-facebook accounts contianer's prompt

### ISSUE THREAD

**Issue by MrwanBaghdad** [mozilla/contain-facebook#96]
I used mutli-account addson to sign in with multiple facebook accounts. 
Since, I installed contain-facebook any request to facebook.com get hijacked by the facebook container. Previously, it would ask me if what container to use.
[I installed the contain-facebook since it opens the new links in a "no container" tab](https://github.com/mozilla/multi-account-containers/issues/847l). Unlike, the multi-acconunt addson. 

Expected behavior will be that a link rule to a given container is only triggered from the the default container (No container). That if I opened a new tab in work container and browsed to facebook.com a prompt is displayed asking what container to use.

**Comment by SoftVision-PaulOiegas** [comment]
Hi @MrwanBaghdad and thanks for your report. Sadly I think you got it a bit wrong. Facebook Container has Multi Account Container at his roots, but functions a bit differently. Facebook Container automatically isolates all the Facebook links into its container to guard you from any tracking that Facebook does on the other pages you visit in normal tabs.

You can read more about how it works on the "[About this extension](https://addons.mozilla.org/en-US/firefox/addon/facebook-container/)" section from the install page.

**Comment by MrwanBaghdad** [comment]
@SoftVision-PaulOiegas Thanks for your reply. I am aware of the relations between mutli-account add-on and the facebook container add-on. 
That's why I think their behaviour shouldn't be conflicting. As I stated above the contain-facebook prevents the multi-account add-on to issue the promopt asking users if they would like to continue to facebook using the "facebook container" or another container which is used to login with another facebook account. That directly leads that one of the multi-account add-on marketed features/use cases, using more than one facebook account, to be unusable.

**Comment by SoftVision-PaulOiegas** [comment]
True, that's a downside of the Facebook-Container add-on. But this is a cost that you have to pay for the extra protection it provides. It forces all the Facebook pages in the same container for enhanced tracking protection. 
If you still want to use multiple Facebook accounts, I think using just Multi Account Containers would be better for your style. You can create customized containers for your Facebook accounts only. It will kind of replicate the protection of Facebook-Container if you don't open other websites in those containers.

**Comment by stoically** [comment]
A workaround here is to navigate to "www.facebook.com" instead of just "facebook.com". In this case Facebook Container detects the MAC Assignment and doesn't interfere

**Comment by MrwanBaghdad** [comment]
@stoically  Sorry if my question seemed dumb. But what is a MAC assignment? Do you mean like MAC addresses?

**Comment by stoically** [comment]
@MrwanBaghdad My bad, throwing around acronyms isn't always helpful. :grin: It just means assigning websites to "Always open in" with Multi-Account Containers

**Comment by jtagcat** [comment]
What about an option 'open in fb container, if opened in the default container' or 'open in fb container, if not open in these containers (allowing you to say what containers are meant for multi-accounts)'?
